### DISCUSSION TITLE
Coordinator Backup, Restore and Migration

### DISCUSSION THREAD

**Discussion by TomKassel** [Koenkk/zigbee2mqtt#24273] [General]
So far I have used z-wave and are now switching the system to Zigbee.
I would like to change the coordinator - from Conbee 2 to Sonoff ZBDongle-E (V2).
At the moment I have 165 devices that I don't want to repair under any circumstances! 
Now the question is why there is no solution in Zigbee2MQTT for backup, restore and migration the coordinator.
 How is it possible that such an important feature is not included? Or have I overlooked something?

**Comment by osgger** [comment]
Maybe this might help (or not, in your case): [https://www.zigbee2mqtt.io/guide/faq/#what-does-and-does-not-require-repairing-of-all-devices](url)

But what has Z-Wave to do with this?

**Reply by TomKassel** [reply]
Thank´s a lot... but the link doesn't work for me
I just wanted to say that I've used z-wave so far and don't have that much experience with Zigbee yet.

**Comment by osgger** [comment]
And Conbees don't seem to have the ability to create a backup of the coordinator:

[https://www.zigbee2mqtt.io/guide/usage/mqtt_topics_and_messages.html#zigbee2mqtt-bridge-request-backup](url)

**Comment by sigma415** [comment]
Here the working link:
https://www.zigbee2mqtt.io/guide/usage/mqtt_topics_and_messages.html#zigbee2mqtt-bridge-request-backup

Is there any possibility to configure the path for the backup file ?
